deploy/basic/terraform/datasources.tf

```diff
@@ -58,4 +71,16 @@ data "oci_core_images" "compute_images" {
   shape                    = var.instance_shape
   sort_by                  = "TIMECREATED"
   sort_order               = "DESC"
-}
\ No newline at end of file
+}
+
+data "template_file" "mushop" {
+  template = "${file("./scripts/node.sh")}"
+}
+
+locals {
```

Comment:
nit: Move `locals` to a single location or have them dispersed to the locations where they are used. 
This one is used in `compute.tf`, but defined here. There are locals in `policies.tf`, but used within `policies.tf`

Reply:
The locals on "datasources" are in a central location, with locals used by multiple resources. The locals in policies.tf are just used by policies.tf.

Reply:
locals moved per recommendation